modules/openstack/topology/common-openstack.tf

```diff
@@ -0,0 +1 @@
+../common-openstack.tf
```

Comment:
topology is no longer a Terraform step. We've collapsed everything down to infra and bootstrap (with plans to further collapse it into a single step).

Reply:
This is the OpenStack's "vpc" module. VPC is not a term that exists in OpenStack so we've called it "topology" but it serves the same purpose.